src/acreplace.rs

```diff
@@ -0,0 +1,38 @@
+use aho_corasick::AhoCorasickBuilder;
+use aho_corasick::AhoCorasick;
+use std::{
+    cell::RefCell,
+    collections::hash_map::HashMap
+};
+
+struct Replacements {
+    pub automaton: AhoCorasick,
+    pub replacements: Vec<String>
```

Comment:
```suggestion
    pub replacements: Vec<String>,
```

Rustfmt should catch these, but I don't think we currently CI enforce it (though I should some day)